cloudinstall/core.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, ui, config, loop):
         self.maas = None
         self.maas_state = None
         self.nodes = None
-        self.charm_modules = utils.load_charms()
+        self.charm_modules = utils.load_charms(
+            self.config.getopt('charm_plugins'))
```

Comment:
I was looking at that.. I don't think so, ill take a closer look and see if we can remove it